kore/src/Kore/Repl/Interpreter.hs

```diff
@@ -1392,7 +1392,7 @@ prettyClaimStateComponent transformation omitList =
             }
   where
     prettyComponent =
-        unparseToString . OrPattern.toTermLike
+        unparseToString . OrPattern.toTermLike _
```

Comment:
@andreiburdusa Where should we get the sort here?

Reply:
Instead of adding a sort, maybe we can use `unparseAssoc'`?

Reply:
Oh, this is just pretty-printing for the user, right? The frontend doesn't need to be able to parse the output?

Reply:
I think it's just for the user since `stuckTransformer` contains the string `"Stuck: \n"` and `provenValue` contains `"Proven."`. But I'm not 100% sure.

Reply:
Could you clarify how `unparseAssoc'` should be used here? For example, I am unsure what is the operator in this case?

Reply:
I actually just implemented this on another branch because I needed it there, too. Let me push it to its own pull request, and we can just use `show . Pretty.pretty` here instead of `unparseToString . OrPattern.toTermLike _`.

Reply:
> I think it's just for the user since `stuckTransformer` contains the string `"Stuck: \n"` and `provenValue` contains `"Proven."`. But I'm not 100% sure.

Those get sent to different "output streams" in kore-repl (we separate kore output and auxiliary output, so that we can send kore output to the frontend). So we actually need the kore here to be correct kore, this function prints the configuration which is what people usually send to the frontend to get unparsed back to K.

Reply:
Then my first idea is to associate sorts to claims. I see we now discard the sorts in `extractClaim` in places like https://github.com/kframework/kore/blob/a6bf9b795b2bcc106329e0fa8b235e9d2559836b/kore/src/Kore/Reachability/OnePathClaim.hs#L188

Reply:
We can just use `SortVariableSort (SortVariable "R")` because the sort will only be used if the `OrPattern` is `\bottom`, and the frontend ignores the sort of `\bottom`.